dask/array/tests/test_array_core.py

```diff
@@ -918,6 +918,29 @@ def test_elemwise_dtype():
             assert da.maximum(x, y).dtype == da.result_type(x, y)
 
 
+@pytest.mark.parametrize(
+    "x,o",
+    [
+        [
+            [11, 2, 3, 4, 5, 12, 34, 56, 78, 90],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+        ],
+        [
+            [[11, 2, 3, 4, 5], [6, 7, 8, 9, 0], [12, 34, 56, 78, 90]],
+            [[1, 1, 1, 1, 1], [1, 1, 1, 1, 1], [1, 1, 1, 1, 1]],
+        ],
+    ],
+)
+def test_elemwise_return(x, o):
```

Comment:
A better name might be something like `test_handle_out_return`; `elemwise` isn't the only user of `handle_out`

Reply:
ohh i searched in the file and saw it is used only once. Now, I saw it is used in different files.